lib/core/include/rodsUser.h

```diff
@@ -22,7 +22,16 @@
 #define PUBLIC_USER_AUTH        1       /* not authenticated yet */
 #define REMOTE_USER_AUTH        2       /* authenticated as remote user */
 #define LOCAL_USER_AUTH         3       /* authenticated as local user */
+// Authenticate as a privileged remote user. This authentication level is reserved for iRODS
+// users from a remote zone which have been given administrative privileges in the local zone.
+// The authenticated privileged remote user can perform actions on behalf of a user from the
+// same remote zone in the local zone. Users with this level of authentication are not allowed
+// to perform actions on behalf of users from other zones and cannot perform certain privileged
+// actions in the local zone, which would require authentication level LOCAL_PRIV_USER_AUTH.
```

Comment:
Any thoughts on these words? I didn't do the other auth levels as I was only dealing with the last two, but I can put words for the others if desired.

Reply:
should the first word be "Authenticated" instead of "Authenticate"?

these values are after authentication has been done, correct?

Reply:
other than that, i think it is very clear.

Reply:
Ah, yes. I think that's right. I'll change the word to past tense.